packages/core/plugins/core-plugins-server-internal/src/discovery/plugins_discovery.test.ts

```diff
@@ -153,8 +151,8 @@ const packageMock = {
   },
 };
 
-const manifestPath = (...pluginPath: string[]) =>
-  resolve(KIBANA_ROOT, 'src', 'plugins', ...pluginPath, 'kibana.json');
+const pluginDir = (...segments: string[]) => resolve(REPO_ROOT, '../kibana-extra', ...segments);
```

Comment:
This is looking at a directory outside the repo, right?

Should we point to a tmp folder instead?

Reply:
It is, and always has. I don't think we need to change this now.

Reply:
I misunderstood what was going on here, this is all virtual file system stuff and the `plugins` dir at the root of the repo is once again in the pluginDirs list so we can just use that. Fixed in 65b61b0d2fa